requirements.txt

```diff
@@ -16,3 +16,12 @@ statsmodels==0.11.1
 tables==3.6.1
 umap-learn==0.4.6
 xarray==0.12.3
+Twisted~=20.3.0
+certifi
+pyOpenSSL
+service_identity
+git+https://github.com/vanvalenlab/kiosk-client@master
```

Comment:
I think the client is pip installable: can you try adding kiosk_client to the requirements.txt file?

It doesn't look like certifi, pyOpenSSL, or service_identify are imported anywhere, is there a reason these are required?

Reply:
I'd be happy to add more methods. 
regarding the requirements - they are required in order to be able to upload files using twisted and for the tls certificate (even tough they are not explicitly called in the method). I'm changing the git dependency to pip installation.
Thanks!